generator/src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java

```diff
@@ -20,11 +18,40 @@
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.algorand.sdkutils.utils.*;
 
-public class Generator {
+public class OpenApiParser {
     public static final String TAB = "    ";
 
     protected JsonNode root;
     protected Publisher publisher;
+    protected final boolean legacyMode;
```

Comment:
let's name it javaGeneration. 'legacy' is not informative and confusing.

Reply:
Sounds good, I'll rename it in my other PR.